apps/t_shirt_delivery/templates/registration/change_tshirt.html

```diff
@@ -24,72 +24,72 @@
 
 
 <script>
-    let unique_years = [];
-    let years_and_options = {};
-    let years_and_tshirt_url = {}
-    let selected_year = null
-    let current_button = null
+    let uniqueYears = [];
+    let yearAndOptions = {};
+    let yearsAndTshirtUrl = {};
+    let selectedYear = null;
+    let currentButton = null;
     {# All T-Shirts from the view: filtered by current user_attendance campaign #}
-    let all_ts = {{ all_tshirts | safe }}
+    let allTshirts = {{ all_tshirts | safe }};
 
     {# Get one Tshirt for each year #}
-    for (let i = 0; i < all_ts.length; i++) {
-        let t_shirt_name = all_ts[i]["fields"]["name"]
-        let t_shirt_preview = all_ts[i]["fields"]["t_shirt_preview"]
+    for (let i = 0; i < allTshirts.length; i++) {
+        let tShirtName = allTshirts[i]["fields"]["name"]
```

Comment:
Still missing semicolon on the end of the line, check all of your code lines, please.